db/tasks.go

```diff
@@ -59,16 +59,18 @@ type TaskInfo struct {
 }
 
 type TaskFilter struct {
-	UUID         string
-	SkipActive   bool
-	SkipInactive bool
-	OnlyRelevant bool
-	ForOp        string
-	ForTenant    string
-	ForTarget    string
-	ForStatus    string
-	ForArchive   string
-	Limit        int
+	UUID              string
+	SkipActive        bool
+	SkipInactive      bool
+	OnlyRelevant      bool
+	ForOp             string
+	ForTenant         string
+	ForTarget         string
+	ForStatus         string
+	ForArchive        string
+	Limit             int
+	CreatedDate       int64
```

Comment:
Instead of using the `Date` suffix, we normally use `At`, to signify that its a timestamp.